### PR TITLE
Sign in upp components

### PR DESCRIPTION
**Pull request by ttugba**
PTAL

Screen Shot 2020-08-12 at 12 53 13 AM
Screen Shot 2020-08-12 at 12 53 41 AM

### CODE REVIEW

**Comment by basil-kawak** [comment]
@gbilgedal @maher-suleyman can you please have a look here!!

File: src/components/signInUp/index.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  font-family: Aktiv Grotesk Corp;
+  font-size: 18px;
+  color: #6e6e6e;
+}
+.authIcon {
+  width: 2rem;
+  margin: 8px;
+  cursor: pointer;
+}
+.leftSection h6 {
```
Comment (louisrli) [inline src/components/signInUp/index.css:82]: pls put newlines between every rule

Comment (louisrli) [reply src/components/signInUp/index.css:82]: eg after line 68, 75, etc

File: src/components/signInUp/SignInUp.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./index.css";
+import SignIn from "./sections/SignIn";
+import SignUp from "./sections/SignUp";
+import { useState } from "react";
+const SignInUp = () => {
```
Comment (louisrli) [inline src/components/signInUp/SignInUp.jsx:10]: newline aboev this line

File: src/components/signInUp/SignInUp.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  const [logging, setLogging] = useState("signing-in");
```
Comment (louisrli) [inline src/components/signInUp/SignInUp.jsx]: for string constants like these, we usually make them all caps

"SIGNING_IN"
"SIGNING_UP"

change here and its usages

File: src/components/signInUp/sections/SignUp.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+import FacebookAuth from "../../../containers/auth/FacebookAuth";
+import GoogleAuth from "../../../containers/auth/GoogleAuth";
+import { Row, Col } from "react-bootstrap";
+
+const SignUp = (props) => {
+  return (
+    <Row className="signinUpContainer justify-content-md-center">
+      <Col className="d-flex mt-5 mb-5 shadow-lg rounded p-0 " lg="7" sm="12">
+        <Col lg="5" sm="5" className="rightSection">
+          <p>need help?</p>
```
Comment (louisrli) [inline src/components/signInUp/sections/SignUp.jsx]: should this line and line 19 be capitalized? or lowercase intentionally

File: src/containers/auth/EmailPasswordSignIn.jsx [hunk 5]
```diff
@@ -19,32 +19,30 @@ const EmailPasswordSignIn = (props) => {
...
+          <Form.Label>Password</Form.Label>
+          <Form.Control
+            type="password"
+            onInput={(e) => {
+              setPassword(e.target.value);
+            }}
+          />
+        </Form.Group>
+        <a className="forgetPassword" href="#">
+          Forget Your Password?
```
Comment (louisrli) [inline src/containers/auth/EmailPasswordSignIn.jsx]: Forgot your password?

**Review by louisrli — COMMENTED** [review]
can one of your teammates review please

**Comment by basil-kawak** [comment]
@maher-suleyman @gbilgedal

**Review by gbilgedal — APPROVED** [review]

**Comment by ttugba** [comment]
PS: these checks will hopefully not fail after we merge, because there is the function.

**Comment by ttugba** [comment]
PTAL @louisrli

File: src/containers/auth/EmailPasswordSignIn.jsx [hunk 6]
```diff
@@ -1,50 +1,159 @@
...
 const EmailPasswordSignIn = (props) => {
   const [email, setEmail] = React.useState("");
   const [password, setPassword] = React.useState("");
+  const [confirmModalShow, setConfirmModalShow] = React.useState(false);
+  const [resetSuccessModalShow, setResetSuccessModalShow] = React.useState(
+    false
+  );
+
+  const validateEmail = (email) => {
+    var reEmail = /^(?:[\w\!\#\$\%\&\'\*\+\-\/\=\?\^\`\{\|\}\~]+\.)*[\w\!\#\$\%\&\'\*\+\-\/\=\?\^\`\{\|\}\~]+@(?:(?:(?:[a-zA-Z0-9](?:[a-zA-Z0-9\-](?!\.)){0,61}[a-zA-Z0-9]?\.)+[a-zA-Z0-9](?:[a-zA-Z0-9\-](?!$)){0,61}[a-zA-Z0-9]?)|(?:\[(?:(?:[01]?\d{1,2}|2[0-4]\d|25[0-5])\.){3}(?:[01]?\d{1,2}|2[0-4]\d|25[0-5])\]))$/;
```
Comment (louisrli) [inline src/containers/auth/EmailPasswordSignIn.jsx]: i don't think you need this, bootstrap email forms come with validation

give the demo a try at the very top of the page:
https://getbootstrap.com/docs/4.0/components/forms/

**Comment by ttugba** [comment]
PTAL

File: src/containers/auth/EmailPasswordSignIn.jsx (hunk 6 above)
Comment (ttugba) [reply src/containers/auth/EmailPasswordSignIn.jsx]: ok, omitted it. can you please take a look again? @louisrli

**Review by louisrli — APPROVED** [review]

**Pull request merged**